src/components/SimpleMessagingCard.tsx

```diff
@@ -12,6 +13,7 @@ export interface Props {
   priority: number
   showOnHomeScreen?: boolean
   testID?: string
+  type: NotificationType
```

Comment:
I wonder why we introduce this prop? Looks like it is not used within the component.

Reply:
ah good question 😅 https://github.com/valora-inc/wallet/blob/58869ca6a0ebe70ab332691ebd85efaa3a871860/src/home/NotificationBox.tsx#L136 <- here is the culprit, and i was too lazy to untangle the types. i've updated now in https://github.com/valora-inc/wallet/pull/4191/commits/188bfdaecb77ea7cb24bf2ffa83425b26f909482, so hope this looks less weird